custom_model_runner/datarobot_drum/drum/args_parser.py

```diff
@@ -703,6 +703,12 @@ def _reg_arg_triton_server_access(*parsers):
                     GPU_PREDICTORS.ALL
                 ),
             )
+            parser.add_argument(
+                ArgumentsOptions.MULTI_CONTAINER_DEPLOYMENT,
+                action="store_true",
+                default=False,
+                help="When true, a model is expected to run in a separate container.",
+            )
```

Comment:
Can you use the term `sidecar` instead in your naming (https://kubernetes.io/docs/concepts/workloads/pods/sidecar-containers/)